src/util/utils.js

```diff
@@ -623,6 +623,30 @@ export function makeCurrencyCodeTable(currencyConfigMap: CurrencyConfigMap): (cu
   return currencyCode => map.get(currencyCode.toLowerCase()) ?? []
 }
 
+export const shuffleArray = <T>(array: T[]): T[] => {
```

Comment:
When using a "known" algorithm like Fisher-Yates over here, you might want to link to an explanation either the Wiki - https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle or at least to the stackoverflow that has this answer - https://stackoverflow.com/a/2450976

Not a "showstopper" but just thought it would be nice to have